lib/fun_ruby/modules/string/strip.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module FunRuby
+  module Modules
+    # @private
+    module String
+      # Removes leading and trailing whitespaces from a string
+      #
+      # @since 0.1.0
+      #
+      # @param string [::String]
+      #
+      # @return [::String]
+      #
+      # @example Basics
+      #   F::Modules::String.strip("  hello  ") #=> "hello"
+      #
+      # @example Curried
+      #   curried = F::Modules::String.strip
+      #   curried.("  hello  ") #=> "hello"
+      #
+      # @example Curried with placeholder
+      #   curried = F::Modules::String.strip(F._)
+      #   curried.("  hello  ") #=> "hello"
```

Comment:
I would add a couple of other examples with `\n` and `\t` at the beginning and at the end